src/sig/example_sig.c

```diff
@@ -30,7 +30,7 @@ static OQS_STATUS example_stack() {
 
 	uint8_t public_key[OQS_SIG_qTESLA_I_length_public_key];
 	uint8_t secret_key[OQS_SIG_qTESLA_I_length_secret_key];
-	uint8_t message[MESSAGE_LEN];
+	uint8_t message[MESSAGE_LEN + OQS_SIG_qTESLA_I_length_sig_overhead];
```

Comment:
Would this be a clearer example if we had separate `message` and `opened_message` variables, where `message` remains of length `MESSAGE_LEN` and is only used as input to `sign`; `opened_message` would be of length `MESSAGE_LEN` + `...overhead` and would only be used as the output to `sign_open`?